agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/telemetry/TelemetryClient.java

```diff
@@ -159,6 +159,8 @@ public void trackAsync(TelemetryItem telemetryItem) {
       throw new AssertionError("telemetry item is missing time");
     }
 
+    // TODO (heya) once we migrate performance counters and jvm metrics to OpenTelemetry, we can
+    // remove this
     if (quickPulse != null) {
       quickPulse.add(telemetryItem);
     }
```

Comment:
I don't think this code is needed, because we don't get quickpulse metrics from trackMetrics anyways (since those are only once a minute)